src/routes/AccountDetailsContainer.js

```diff
@@ -126,7 +127,7 @@ class AccountDetailsContainer extends React.Component {
     const patronGroup = this.getPatronGroup();
     const itemDetails = this.getItemDetails();
 
-    if (!account) return (<ViewLoading defaultWidth="100%" paneTitle="Loading accounts" />);
+    if (!account) return (<LoadingView defaultWidth="100%" paneTitle="Loading accounts" />);
```

Comment:
`Loading accounts` should probably be i18ned while we're here. There's multiple cases with `LoadingView` throughout the code that could be handled here.